torch/utils/data/_utils/collate.py

```diff
@@ -63,6 +63,8 @@ def default_collate(batch):
             numel = sum(x.numel() for x in batch)
             storage = elem.storage()._new_shared(numel)
             out = elem.new(storage)
+            # Resize 'out' before passing it to 'stack'
+            out = out.resize_((len(batch), *elem.shape))
```

Comment:
After this change, the CI is now passing. In other words, I think `cat` and its variants don't seem to be raising warnings internally anymore. Thus, I will remove the `expectedFailure` from all of them: `cat`, `stack`, `hstack`, `vstack`, and `dstack`.

Reply:
Hi, @ysiraichi - I think you can also remove the skip from `column_stack` in this PR after this change since it also seems to be calling `hstack`. If you already had it noted, please ignore. :) Thank you for the PR.